cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -128,6 +135,24 @@ open class Node @Inject constructor(private val project: Project) {
             }
         }
 
+    /**
```

Comment:
Similarly, are we supporting each node having its own `dockerImage`? Or could this property also be moved to `Dockerform` so that all nodes are forced to share the same one?

Reply:
At the moment even though every node has a dockerimage property, all use the same image. Moving this into Dockerform for now